postcss.config.cjs

```diff
@@ -14,17 +14,16 @@ module.exports = {
     postcssPresetEnv({
       stage: 0,
       autoprefixer: false,
+      enableClientSidePolyfills: false,
```

Comment:
Fairly recent plugin option that disables all plugins that need client side JS to function (`:has`, `:focus-within`, ...)